src/Text/Libyaml.hs

```diff
@@ -253,37 +305,24 @@ getEvent er = do
         YamlScalarEvent -> do
             yvalue <- c_get_scalar_value er
             ylen <- c_get_scalar_length er
-            ytag <- c_get_scalar_tag er
-            ytag_len <- c_get_scalar_tag_len er
-            ystyle <- c_get_scalar_style er
-            let ytag_len' = fromEnum ytag_len
             let yvalue' = castPtr yvalue
-            let ytag' = castPtr ytag
             let ylen' = fromEnum ylen
             bs <- packCStringLen (yvalue', ylen')
-            tagbs <-
-                if ytag_len' == 0
-                    then return Data.ByteString.empty
-                    else packCStringLen (ytag', ytag_len')
```

Comment:
Heh, doh, now I realize why you have it set up this way with the len, I did it that way in the first place. I _think_ my claim that we can use `packCString` instead still applies, but your call on whether to include that now or if I try it separately after this gets merged.